src/components/Metrics.js

```diff
@@ -0,0 +1,54 @@
+import { React, useEffect } from 'react';
+import { useDispatch, useSelector, shallowEqual } from 'react-redux';
+import { NavLink } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faArrowAltCircleRight } from '@fortawesome/free-solid-svg-icons';
+import { getDataFromApi } from '../redux/metric-redux';
+
+const Metrics = () => {
```

Comment:
- [ ] In your Homepage kindly include a filtering mechanism, like a search box that users can use to filter countries on the home page.

  ![wuye-1](https://user-images.githubusercontent.com/81468675/175039619-cdab013d-6464-4251-bddb-2c3e13f0d2a7.jpg)